docs/docusaurus/docs/conceptual_guides/gx_overview.md

```diff
@@ -0,0 +1,148 @@
+
+- **Data Context:** Manages the settings and metadata for a GX project, as well as providing an entry point to the GX Python API.
+- **Datasources:** tell GX how to connect to your source data, and how to organize the retrieved data for future use.
+- **Expectations:** tell GX the standards to which your data should conform.
+- **Checkpoints:** validate a set of Expectations against a specific set of data.
+
+
+## Data Context
+
+![A Data Context](./images_overview/data_context_flowchart.png)
+
+A Data Context defines a GX project.  In Python, the Data Context object serves as your entry point for the GX API and manages various classes to limit the objects you need to directly manage yourself.  A Data Context contains all of the metadata used by GX, the configurations for GX objects, and the output from validating data.
+
+The following are the available Data Context types:
+- **Ephemeral Data Context:** Exists in memory, and does not persist beyond the current Python session.
+- **File Data Context:** Exists as a folder and configuration files. Its contents persist between Python sessions.
+- **Cloud Data Context:** Supports persistence between Python sessions, but additionally serves as the entry point for Great Expectations Cloud.
+
+- [Data Context tutorials](/docs/guides/setup/configure_data_contexts_lp)
+
+### The GX API
+
+A Data Context object in Python provides convenience methods to further configure and interact with GX.  These methods and the objects and additional methods accessed through them compose the GX public API.  
+
+Through the GX public API you will be able to connect GX to your data sources, define the standards to which you expect your data to conform, retrieve data from your data sources, and validate that data against your defined standards.  When working with GX in Python, you will almost always start by instantiating an existing or new Data Context with which to access the GX public API.
+
+- [The GX API reference](/docs/reference/api_reference)
+
+### Settings and Stores
+
+Stores contain the metadata GX uses.  This includes things such as configurations for GX objects, information that is recorded when GX validates data, and credentials used for accessing data sources or remote environments.  GX utilizes one Store for each type of metadata, and the Data Context contains the settings that tell GX where that Store should reside and how to access it.
+
+A Data Context includes default configurations for your Store locations. The default location of Stores in a Filesystem or Cloud Data Context is nested within the same folder hierarchy that contains the Data Context itself. An Ephemeral Data Context will default to keeping your Stores in memory. You can update these defaults to specify other storage locations, such as hosting them in a shared cloud environment or external folders.
+
+- [Stores tutorials](/docs/guides/setup/setup_overview_lp)
+
+### Data Docs
```

Comment:
can we maybe use #### for these to make it more visually obviously that these concepts belong to Data Context?

Reply:
I don't really like jumping heading levels like that.  It would probably be better to update our CSS so that the change is throughout the docs.  I think that would be a separate PR, though, since the change would be docs-wide.

Reply:
ok agreed